### PR TITLE
Add playground project

### PR DESCRIPTION
**Pull request by nohwnd**
## Description
Add playground project that is setup for one end-to-end flow, that can be debugged under VS directly without rebuilding and re-packing the whole solution all the time. To keep it simple I am using .NET Framework, because that does not need microsoft.net.test.sdk installed.

### CODE REVIEW

**Review by Haplois — APPROVED** [review]

**Review by Haplois — APPROVED** [review]
Nit: I think we should add a Readme file to this folder to document what it does.

LGTM

File: README.md [hunk 1]
```diff
@@ -1,58 +1,17 @@
-### VSTest
```
Comment (MarcoRossignoli) [inline README.md:1]: I think this is the main readme right? Should we put the new one inside playground folder?

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs [hunk 2]
```diff
@@ -71,7 +71,7 @@ public string LogFilePath
                 var directoryPath = Path.GetDirectoryName(value);
                 if (!string.IsNullOrEmpty(directoryPath) && !fileHelper.DirectoryExists(Path.GetDirectoryName(value)))
                 {
-                    throw new ArgumentException("LogFilePath must point to a valid directory for logging!");
+                    Directory.CreateDirectory(Path.GetDirectoryName(value));
```
Comment (MarcoRossignoli) [inline src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs:74]: maybe use `directoryPath` everywhere for clarity

**Pull request merged**